Source/WebCore/fileapi/BlobURL.cpp

```diff
@@ -92,7 +92,7 @@ bool BlobURL::isSecureBlobURL(const URL& url)
 URL BlobURL::createBlobURL(const String& originString)
```

Comment:
This should take a `StringView` instead of a `const String&`.

Reply:
Sounds good, changed.